src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,16 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String bullColor;
+    private int bullHeight;
+    private int bullLength;
```

Comment:
It's not a good practice to use such names for class fields (better to name it without `bull` prefix). As you already have `color` field in class Machine you can replace it with more type-specific field

Reply:
The same for other classes